strictdoc/export/html/templates/screens/project_index/project_tree_folder.jinja

```diff
@@ -19,7 +19,7 @@
     {% endfor %}
 
     {% for file_ in folder.files %}
-      {% if file_.has_extension(".sdoc") %}
+      {% if file_.has_extension(".sdoc") or file_.has_extension(".junit.xml") %}
```

Comment:
Double-check that on the single document screen, the reports are also displayed.